packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -226,6 +226,17 @@ class DescopeWc extends BaseDescopeWc {
     const loginId = this.sdk.getLastUserLoginId();
     const flowConfig = await this.getFlowConfig();
     const projectConfig = await this.getProjectConfig();
+    const flowVersions = {
```

Comment:
I would expect this to be in the BaseDescopeWc class

In addition, why do we need here also?
```
version: flowConfig.version,
componentsVersion: projectConfig.componentsVersion,
```

`componentsVersion` is sent as a separate param, 
and flow version should be already included in `allVersions`

Reply:
i wanted in next phase to remove `version` and `componentsVersion` so they will be only in this object. 
But decided to remove now. 

I don't know in all places the current flowID so prefer to have the option to know it without that